### PR TITLE
Add community standup info to docs

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

**Comment by ajcvickers** [comment]
I only did the first few for now, so we can sign-off on the format before doing all the others.

**Review by roji — APPROVED** [review]
Looks great!

Maybe a table/list on top (or on top of every year?) which allows seeing all the titles and linking to each talk?

**Comment by ErikEJ** [comment]
Great idea to give more visibility to all this great content

**Comment by ajcvickers** [comment]
@roji Added a table (or three). Let me know what you think.

File: entity-framework/core/learn-more/community-standups.md [hunk 1]
```diff
@@ -12,8 +12,43 @@ The .NET Data Community Standups are live-streamed every other Wednesday to Twit
...
+
+Each episode is summarized in the table below, with a link to full details.
+
+For EF Core episodes:
+
+- Intro-level episodes usually require only a minimum of experience with EF Core.
+- Intermediate episodes go more deeply into EF Core features.
+- Advanced episodes cover advanced features and/or EF Core internals.
+
+### Intro-level episodes
```
[the remark this answers is not in the document]
Comment (ajcvickers) [reply entity-framework/core/learn-more/community-standups.md:25]: Yeah, I think I agree.

**Comment by SamMonoRT** [comment]
Excellent idea!

**Comment by luisquintanilla** [comment]
Looks great!

**Comment by ajcvickers** [comment]
Updated to include all 74 .NET Data Community Standups.

**Review by roji — APPROVED** [review]
It's really, really nice to see this - and I can imagine that it was quite a bit of work. This should really increase the visibility of our standups...

**Comment by ajcvickers** [comment]
It was a fun trip down memory lane. I had forgotten we did some of these!

**Comment by ajcvickers** [comment]
>  I can imagine that it was quite a bit of work.

Almost all of it on the weekend, so not impacting my _real_ work too much.

**Comment by roji** [comment]
Oh BTW there's needs to be a TOC entry point to this no?

**Comment by ajcvickers** [comment]
Added to TOC

**Pull request merged**